cli/daemon/process_test_helpers.go

```diff
@@ -0,0 +1,86 @@
+package daemon
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"syscall"
+	"time"
+
+	ps "github.com/mitchellh/go-ps"
+)
+
+const (
+	StartTestWorkerMsg = "Test worker started"
+	StopTestWorkerMsg  = "Test worker stopped"
+)
+
+const (
+	TestProcessLogPath = "test_process.log"
+	TestProcessPidFile = "test_process.pid"
+)
+
+// waitProcessStart waits for the new process to set signal handlers.
+func waitProcessChanges() {
+	// We need to wait for the new process (tarantool instance) to set handlers.
+	// It is necessary to update for more correct synchronization.
+	time.Sleep(500 * time.Millisecond)
+}
+
+func cleanupDaemonFiles(logFilename string, pidFilename string) {
+	if _, err := os.Stat(logFilename); !os.IsNotExist(err) {
+		os.Remove(logFilename)
+	}
+
+	if _, err := os.Stat(pidFilename); !os.IsNotExist(err) {
+		os.Remove(pidFilename)
+	}
+}
+
+func readPID(filePath string) (int, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return 0, err
+	}
+
+	buf := bytes.NewBufferString("")
+	if _, err = io.Copy(buf, file); err != nil {
+		return 0, err
+	}
+
+	pid, err := strconv.Atoi(buf.String())
+	if err != nil {
+		return 0, err
+	}
+
+	return pid, nil
+}
+
+func IsDaemonAlive(pid int) (bool, error) {
```

Comment:
Add a description for the public function.

Reply:
Done.